db/migrate/20180912060726_create_schools.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSchools < ActiveRecord::Migration[5.2]
+
+
+  def up
+    create_table :schools do |t|
+      t.string "name"
+      t.string "dean_name"
+      t.integer "position"
```

Comment:
Please define Column attributes as per your model.

Reply:
as it is a migration file, i wont be able to make changes to it